app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :post
   belongs_to :user
+
+  def self.update_like_counter(post_id)
+    post = Post.find_by(id: post_id)
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_like_counter` method every time a new like has been saved to the database, then make the `update_like_counter` method private